doc/en/announce/release-2.0.0.rst

```diff
@@ -62,7 +62,7 @@ New Features
 - new "-q" option which decreases verbosity and prints a more
   nose/unittest-style "dot" output.
 
-- many many more detailed improvements details
+- many, many, more detailed improvements details
```

Comment:
generally in English, doubled words aren't appropriate without punctuation, and often they're just errant.